### PR TITLE
Add Flux config for FPL case service in PROD

### PR DESCRIPTION
**Pull request by geek-soft**
### JIRA link (if applicable) ###

None

### Change description ###

Adds Flux config for FPL case service in PROD

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/prod/common/family-public-law/fpl-case-service.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      image: hmctspublic.azurecr.io/fpl/case-service:prod-493b765c
+      ingressHost: fpl-case-service-prod.service.core-compute-prod.internal
+      environment:
+        JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MinRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0
+          -XX:+UseConcMarkSweepGC -javaagent:/opt/app/applicationinsights-agent-2.5.0-BETA.3.jar
+        IDAM_API_URL: https://idam-api.platform.hmcts.net
+        IDAM_S2S_AUTH_URL: http://rpe-service-auth-provider-prod.service.core-compute-prod.internal
+        CORE_CASE_DATA_API_URL: http://ccd-data-store-api-prod.service.core-compute-prod.internal
+        DOCUMENT_MANAGEMENT_URL: http://dm-store-prod.service.core-compute-prod.internal
+        CCD_UI_BASE_URL: https://www.ccd.platform.hmcts.net
```
Comment (timja) [inline k8s/prod/common/family-public-law/fpl-case-service.yaml]: what do you need this for in your app? emails?

File: k8s/prod/common/family-public-law/fpl-case-service.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      aadIdentityName: family-public-law
+      keyVaults:
+        fpl:
```
Comment (naikajah) [inline k8s/prod/common/family-public-law/fpl-case-service.yaml]: shouldn't this be in double quotes?

File: k8s/prod/common/family-public-law/fpl-case-service.yaml (hunk 1 above)
Comment (geek-soft) [reply k8s/prod/common/family-public-law/fpl-case-service.yaml]: Uri is used to format URLs in emails

File: k8s/prod/common/family-public-law/fpl-case-service.yaml (hunk 2 above)
Comment (geek-soft) [reply k8s/prod/common/family-public-law/fpl-case-service.yaml]: This is simple value, no need for quotes IMO. Same works just fine in other envs.

Comment (timja) [reply k8s/prod/common/family-public-law/fpl-case-service.yaml]: doesn't matter afaik

**Pull request merged**